third_party/xla/tsl/BUILD

```diff
@@ -0,0 +1,55 @@
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Where do you copy this stuff from... 2015?

Reply:
It was copied from TF OSS code.
I think we need to figure out the new copyright text, but let's address it later.